picasso/src/main/java/com/squareup/picasso/BitmapHunter.java

```diff
@@ -446,19 +446,29 @@ static Bitmap transformResult(Request data, Bitmap result, int exifRotation) {
       if (data.centerCrop) {
         float widthRatio = targetWidth / (float) inWidth;
         float heightRatio = targetHeight / (float) inHeight;
-        float scale;
+        float scaleX, scaleY;
         if (widthRatio > heightRatio) {
-          scale = widthRatio;
+          scaleX = widthRatio;
+          scaleY = widthRatio;
           int newSize = (int) Math.ceil(inHeight * (heightRatio / widthRatio));
           drawY = (inHeight - newSize) / 2;
           drawHeight = newSize;
+          int rounded = Math.round(drawHeight * scaleY);
+          if (rounded != targetHeight) {
```

Comment:
Is it worth it to do the round and this comparison?

Why not just always set the `scaleY` to `targetHeight / (float) newSize` (and similar below)?